modules/nw-configure-ingress-access-logging.adoc

```diff
@@ -116,3 +116,46 @@ spec:
   logging:
     access: null
 ----
+
+Allowing IngressController to modify the HAProxy log length when using a sidecar.
```

Comment:
Sorry, but don't see the peer review feedback [here](https://github.com/openshift/openshift-docs/pull/65658#discussion_r1371799047) impemented. Kindly, considering implementing as is makes the instructions parallel with the rest and suggests correcting the object reference. 
You may use either a [general reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references) or use an [object reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references) for object names.

Reply:
@abhatt-rh All comments incorporated, will squash.